components/script/dom/intersectionobserver.rs

```diff
@@ -42,6 +42,11 @@ use crate::script_runtime::{CanGc, JSContext};
 pub(crate) struct IntersectionObserver {
     reflector_: Reflector,
 
+    /// > The root provided to the IntersectionObserver constructor, or null if none was provided.
+    /// <https://w3c.github.io/IntersectionObserver/#dom-intersectionobserver-root>
+    #[ignore_malloc_size_of = "mozjs"]
```

Comment:
We can remove this by adding an entry for the `ElementOrDocument` union to https://github.com/servo/servo/blob/main/components/script_bindings/codegen/Bindings.conf#L607 that derives MallocSizeOf.

Reply:
Thanks! That would make the implementation much more clean.